src/porepy/examples/tracer_flow.py

```diff
@@ -0,0 +1,274 @@
+"""Module containing a simple tracer flow setup, modelled as a single phase, 2-component
+flow."""
+
+from __future__ import annotations
+
+from typing import Literal, Sequence, TypedDict, cast
+
+import numpy as np
+
+import porepy as pp
+from porepy.applications.material_values.fluid_values import water as _water
+from porepy.applications.md_grids.domains import nd_cube_domain
+from porepy.compositional.compositional_mixins import CompositionalVariables
+from porepy.models.compositional_flow import (
+    BoundaryConditionsMulticomponent,
+    ComponentMassBalanceEquations,
+    InitialConditionsFractions,
+)
+from porepy.models.fluid_mass_balance import SinglePhaseFlow
+
+
+class WaterDict(TypedDict):
+    """For mypy when using **water in combination with FluidComponent."""
+
+    compressibility: float
+    density: float
+    specific_heat_capacity: float
+    thermal_conductivity: float
+    thermal_expansion: float
+    viscosity: float
+    name: str
+
+
+# Casting this to help mypy
+water = cast(WaterDict, _water)
+
+
+class DomainX(pp.PorePyModel):
```

Comment:
I don't think this belongs here. We should either reuse the SquareDomainOrthogonalFractures or, if you need immersed fractures, add this to the application folder. Do you agree, @keileg?

Reply:
I also think this could inherit from SquareDomainOrthogonalFractures

Reply:
Simplified the geometry by inheriting from `SquareDomainOrthogonalFractures`. I only overwrite the fractures now to get an X shape

Reply:
I modified the setup slightly, it now uses the original SquareDomainOrthogonalFractures.